posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,615 @@
+* <<bugs, Notable bug fixes>>
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== Run your apps using 23.0.0.3
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>23.0.0.3</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[23.0.0.3,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24758
+// Contact/Reviewer: jhanders34,ReeceNana
+// // // // // // // // 
+[#jakarta10]
+== Jakarta EE 10 Core Profile, Web Profile, and Platform
+Jakarta EE 10 Core Profile, Web Profile and Platform are now officially supported in Open Liberty! We'd like to start by taking the time to thank all those who provided feedback throughout our various betas.
+
+
+Jakarta EE 10 marks a major milestone as it is the first release to provide updates to the specifications since Java EE 8 in 2017 and, therefore, the first release to provide spec updates since being taken over by the Eclipse Foundation. Among the many updates to existing specifications, it also introduces the Core Profile. The Core Profile is specifically aimed at lightweight runtimes, like Open Liberty, that are optimized for running cloud-native Java microservices.
+ 
+Liberty provides convenience features for running all of the component specifications that are contained in the Jakarta EE 10 Web Profile (`link:{url-prefix}/docs/latest/reference/feature/webProfile-10.0.html[webProfile-10.0]`) and Jakarta EE 10 Platform (`link:{url-prefix}/docs/latest/reference/feature/jakartaee-10.0.html[jakartaee-10.0]`).  For Jakarta EE 10 features in the application client, use the `jakartaeeClient-10.0` Liberty feature.
+
+Using these convenience features enables developers to rapidly develop applications using all of the APIs contained in the Web Profile and Platform specifications. If you've previously used the `webProfile-9.1` Liberty convenience feature, note that, unlike the `webProfile-9.1` feature, the `webProfile-10.0` Liberty feature does not enable the XML Binding feature because XML Binding is not a Web Profile component specification. If your application relies on XML Binding APIs, you must explicitly enable the `xmlBinding-4.0` Liberty feature when using the `webProfile-10.0` feature.

```

Comment:
Do we want to just drop these two paragraphs completely?
The first one is largely handled at the end of the Jakarta section, and the second paragraph seems a bit too specific compared the broad scoped nature of the post.